core/src/main/kotlin/org/rewedigital/katana/Component.kt

```diff
@@ -284,18 +309,22 @@ class ComponentContext private constructor(
 ) {
 
     @PublishedApi
-    internal fun <T> injectByKey(key: Key, internal: Boolean = false, arg: Any? = null): T {
+    internal fun <T> findInstance(key: Key, internal: Boolean = false, arg: Any? = null): Instance<T>? {
         val instance = thisComponent.thisComponentInjectByKey<T>(key, internal, arg)
```

Comment:
`thisComponentInjectByKey` shouldn't be `thisComponentFindInstance`?

Reply:
Yes, you're right. Thanks for the find!